src/main/java/bio/terra/common/kubernetes/KubeService.java

```diff
@@ -22,6 +18,9 @@
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicBoolean;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
I get a lot of thrash if I run the intellij 'optimize imports' action, for instance it reverts this change. I'm just using the default google-java-format.

Reply:
IntelliJ and spotless have different opinions about import ordering. Spotless wins, but it does mean the odd change. I would expect that it would resolve after awhile. If we always `spotless apply` on a build, everything we have in github should be in spotless format and we wouldn't get spurious diffs. Right?

Reply:
Occasionally I forget to build before committing changes and pushing my branch!

On the AoU side, we actually do a spotlessDiff as part of our unit tests, so a PR can't merge unless the format is exactly how Spotless would want it. Something we can consider if this becomes a persistent pain point.